### ISSUE TITLE
Add property/properties to represent EIDR identifiers (of media works and edits/editions)

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#2469]
Analogous to /isbn, there is a widely used identifier called [EIDR](https://eidr.org/) ( [wikipedia](https://en.wikipedia.org/wiki/EIDR), [wikidata](https://www.wikidata.org/wiki/Property:P2704) ) which we don't have a property for in schema.org.

I propose fixing this, using the same definition as Wikidata: "Identifier for a film or television work, edit or manifestation, in the Entertainment Identifier Registry".

According to Wikipedia, it is applicable to a wide range of entity types, so I suggest associating the property simply with CreativeWork.

> "The identification system resolves an identifier to a metadata record that is associated with top-level titles, edits, DVDs, encodings, clips, and mash-ups. EIDR also provides identifiers for Video Service providers, such as broadcast and cable networks.

In terms of expected adoption, we would expect to make some use of this at Google.

**Comment by danbri** [comment]
Looking into the details, it seems there are two flavours of EIDR identifier, which share a common identifier space and which can't be distinguished by inspecting the identifier string:

* eidr_title:  Title (Abstraction) Identifier. Abstraction level EIDR or equivalent.
* eidr_edit: Asset Identifier. This should be the Edit Level EIDR

The former identifies larger units of content, the latter is specific to edits. We may need two properties, in which case I suggest attaching them just to audio/visual types so that we don't over-clutter /CreativeWork.

**Comment by danbri** [comment]
I got this far,

```
<div>

  <!-- see issue, may need to break out edits vs titles, in which case let's attach to specific types to avoid /CreativeWork clutter. -->

    <div typeof="rdf:Property" resource="http://schema.org/eidr">
        <span class="h" property="rdfs:label">eidr</span>
        <span property="rdfs:comment">An [[EIDR]](https://eidr.org/) (Entertainment Identifier Registry) [[identifier]] for a film or television work, edit or manifestation.</span>
        <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/CreativeWork">CreativeWork</a></span>
        <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
        <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/URL">URL</a></span>
        <span>Category: <span property="schema:category">issue-2469</span></span>
        <link property="http://schema.org/isPartOf" href="http://pending.schema.org" />
        <span>Source:  <a property="dc:source" href="https://github.com/schemaorg/schemaorg/issues/2469">#2469</a></span>
    </div>

</div>
```

**Comment by WeaverStever** [comment]
Related: I noticed this morning that [MusicRecording](https://schema.org/MusicRecording) does not have the ISWC identifier https://www.wikidata.org/wiki/Property:P1827

The ISWC is the global identifier for the work (this identifies the writer and publisher). The ISRC (which is already a property of MusicRecording (https://schema.org/isrcCode) designates a recording of a work.

For instance, T-Bone Walker's "Stormy Monday" (ISWC = T0702439915) has been recorded hundreds of times by dozens of artists, each of those recordings will (eventually) have a unique ISRC.  

Since the schema for isrcCode mentions Musicbrainz, the collection of recordings related to the Stormy Monday ISWC are here. https://musicbrainz.org/work/cf688650-2a34-3700-aa7d-07157d0a7ec5  The format for ISWC on MusicBrainz is: **ISWC: T-070.243.991-5** and the ISWC is one of the main identifiers for using the MusicBrainz API.

The ISWC contains original publisher and author information for licensing and even an original artist will want to include the code to credit the original publisher.   Additional "publishers" (https://schema.org/publisher) are generally licensed create / publish the recording (a rendition of the work) and pay residuals to the owners listed in the ISWC. 

When music is used in a film, a "sync" license is required, and the ISWC is the modern source for identifying the parties to obtain the sync license.

**Comment by danbri** [comment]
Looking at this a bit more, https://ui.eidr.org/view/content?id=10.5240/7EC7-228A-510A-053E-CBB8-J (click around for full record and relationships) you can see the model is very FRBR-like. They have strong identifiers at the broadest level, aspiring to a single ID for a Work("Abstraction"). Then Edit is the different edits/cuts of a movie. So the first Ghostbusters movie gets a single ID at the top level and a bunch of edits, each of which can also be said to have manifestations. Individual copies/items seem to be out of scope.

**Comment by danbri** [comment]
I think /workEIDR and /editEIDR for the two concepts here... see https://ui.eidr.org/view/content?id=10.5240/B752-5B47-DBBE-E5D4-5A3F-N (relationships tab)

**Comment by danbri** [comment]
See https://github.com/schemaorg/schemaorg/issues/2482#issuecomment-598231351 for attempt at an example.

**Comment by RichardWallis** [comment]
Implemented in commit 60c9674b81b5ba79dff3bd474c7b10c86eaf9fae

**Comment by gmackenz** [comment]
#3297 Requesting update to the [titleEIDR](https://schema.org/titleEIDR) to add it to [TVSeason](https://schema.org/TVSeason) and [TVSeries](https://schema.org/TVSeries) as EIDR identifiers are used for those instances by the EIDR and Google:

E.g.:
TVSeries for "12 Monkeys": https://ui.eidr.org/view/content?id=10.5240/354B-4B1D-02AA-068E-CDFD-3 with titleEIDR 10.5240/354B-4B1D-02AA-068E-CDFD-3 TVSeason for "12 Monkeys: Season 1": https://ui.eidr.org/view/content?id=10.5240/9450-3CB5-BB29-5430-CCDA-B with titleEIDR 10.5240/9450-3CB5-BB29-5430-CCDA-B

Google's Media Action Developer site already recommends titleEIDR for both [TVSeason](https://developers.devsite.corp.google.com/actions/media/reference/data-specification/tv-shows-specification#tvseason) and [TVSeries](https://developers.devsite.corp.google.com/actions/media/reference/data-specification/tv-shows-specification#tvseries).
